ledger/eval/eval.go

```diff
@@ -1339,7 +1345,13 @@ func (eval *BlockEvaluator) TestingTxnCounter() uint64 {
 }
 
 // Call "endOfBlock" after all the block's rewards and transactions are processed.
-func (eval *BlockEvaluator) endOfBlock() error {
+// When generating a block, participating addresses are passed to prevent a
+// proposer from suspending itself.
+func (eval *BlockEvaluator) endOfBlock(participating ...basics.Address) error {
```

Comment:
Why `...basics.Address` instead of `[]basics.Address`? I assume callers always have a slice, as opposed to call sites with, say, 5 explicit arguments.